locales/fr.json

```diff
@@ -45,8 +45,7 @@
   "phase.desc": "This is a test site and should NOT be used to apply for this service.",
```

Comment:
This needs updating to something like "Ce site changera au fur et à mesure que nous testerons les idées."

Reply:
Good find! Updated with that message.